### ISSUE TITLE
All emails imported in april are missing an attachment

### ISSUE THREAD

**Issue by exander77** [ProtonMail/proton-bridge#86]
I am reimporting emails with Import-Export tool and I noticed that one of my emails lost one of its attachment. Attachment is just not there and after download, the email has 5,7MB instead of 11,4MB.

As there is no way to remove attachments from Web UI, I would really like to know how this happened.

**Comment by exander77** [comment]
Found another one...

**Comment by exander77** [comment]
And another one, wtf?

**Comment by exander77** [comment]
![Screenshot_2020-10-03_21-54-34](https://user-images.githubusercontent.com/2256039/95000716-7551f880-05c3-11eb-824a-ab7861cddaa7.png)

**Comment by exander77** [comment]
No, this is not random. Every email I imported in April is missing one achievement.

This I want to hear some really good fucking explanation.

**Comment by jameshoulahan** [comment]
We have an issue template. Use it. Spammy reports waste everyone's time.

Version?

**Comment by exander77** [comment]
There is nothing spammy, what is wrong with you? Up to date version, which was available on 22nd April 2020 (this is when I did the migration).

Do you understand that you destroyed an attachment in every single email I imported?

**Comment by jameshoulahan** [comment]
> There is nothing spammy, what is wrong with you?
I'm kindly requesting you to open issues that follow the template we have in place. Failure to do so makes it hard to follow the thread.

Perhaps you could consider trying the beta version of the Import-Export app?

**Comment by exander77** [comment]
@jameshoulahan I am currently testing the latest version of Import-Export tool, is there a beta version? Where is it available?

**Comment by jameshoulahan** [comment]
Yes, you can find a beta version [here](https://protonmail.com/download/beta/ie/protonmail-import-export-app_1.1.1-1_amd64.deb).

**Comment by jameshoulahan** [comment]
As mentioned in #89, I think it's best if we postpone discussion until you have tried to reproduce the issue on the new beta version of the app. Deep-diving into issues caused by code which no longer exists will only lead to confusion.

**Comment by andrzejsza** [comment]
@exander77 I know that you still have some issues with the parser, but seems that the missing attachment issue is now fixed.
